pylib/mozhginfo/mozhginfo/push.py

```diff
@@ -0,0 +1,130 @@
+class Push(object):
+    """This represents a Push"""
+
+    def __init__(self, **kwargs):
+        """
+            We set Push object default as dict, and we will
+            give each push a push id when we init it.
+        """
+        self._id = kwargs.get('push_id', None)
+        push_info = kwargs.get('push_info', None)
+        if push_info:
+            self._push = dict()
+            self._push['changesets'] = []
+            self._push['date'] = push_info.get('date', None)
+            self._push['user'] = push_info.get('user', None)
+            for changeset in push_info.get('changesets'):
+                # we need to handle the situation that only has node number.
+                if isinstance(changeset, basestring):
+                    self._push['changesets'].append(Changeset(node=changeset))
```

Comment:
Could you please explain this a bit more? Add a more complete note so people can understand what you mean.